tutorial_src/puz/prog/puz.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+
+ int main() {
+    std::ios_base::sync_with_stdio(false);
+    std::cin.tie(nullptr);
```

Comment:
Chcemy magiczne linijki w wzorcówce?

Reply:
No imo chcemy, często się je daje to czemu mielibyśmy zapomnieć nagle że istnieją.

Reply:
Można też dopisać komentarz, czemu one tu są. Np. podlinkować https://oi.edu.pl/l/oi_przyklady_zadan/

Reply:
No to mi wydaje się że linkowanie stron olimpiady brzmi jak średni pomysł, nigdzie tego nie robimy a moim zdaniem to jakoś wybitnie nie tłumaczy dlaczego się te linijki daje.
Równie dobrze możemy napisać "ze względów wydajnościowych".

Prócz tego dużo różnych rzeczy związanych z samym cpp nie tłumaczymy np jakichś ifstreamów przy checkerkach, więc dlaczego tłumaczyć to.